src/http_client/http_client.ts

```diff
@@ -102,8 +102,11 @@ class HttpClientRequestDecoration {
    */
   create(options?: HttpClientOptions): HttpClient {
     return this.appHttpClient.extend({
-      headers: { [this.correlationIdHeader]: this.correlationId },
       ...options,
+      headers: {
```

Comment:
Nur eine Frage: ist es egal, ob `headers:` vor `...options` oder danach ist? I.e. im Zeile 105 oder 106?

Reply:
It does matter, because `options` contains a property named `headers` and the latter would be overwritten if we wouldn't change the order

Reply:
that was actually the bug :-D